app/views/accounts/register.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Please register below</h1>
+<% if @errors %>
+<% errors.each do |message| %>
+  <%= message %>
+<%end%>
+<%end%>
+
+<%= form_for @user, url: {action: "create"}, html: {class: "nifty_form"} do |f| %>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :email %>
+  <%= f.text_field :email %>
+  <%= f.label :password %>
+  <%= f.password_field :password %>
```

Comment:
It's good practice to also add a `password_confirmation` label and field here. `has_secure_password` will deal with it if it's there.